data/source.py

```diff
@@ -0,0 +1,20 @@
+import csv
+
+from questions import ExampleQuestion
```

Comment:
ExampleQuestion seems is not have too many references to make you using from import statement, on the other hand, if a type would be using so much by another module than maybe it should be moved to the other module